src/UpdatedPackagesCollector.js

```diff
@@ -151,6 +151,7 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
+    // TODO: Ugh... something about this.
```

Comment:
Yeah, or even just config at the outer level.

Like:

```json
{
  "asini": "x.x.x",
  "version": "1.2.0",
  "commands": {
    "publish": {
        "ignore": "test-package"
    },
  },
  "ignore": "irrelevant-package"
}
```

But that still doesn't help with _groups_ of commands that should behave similarly.

Like, I _want_ `bootstrap` to run on `test-package`, but I don't want `publish` to run on it, and I want `updated` to reflect what `publish` will do...

Maybe the answer is `updated` is really just `publish --dry-run`?